.cbmc-batch/include/bn_utils.h

```diff
@@ -16,6 +16,7 @@
 #ifndef BN_UTILS_H
 #define BN_UTILS_H
 
+#include <stdbool.h>
```

Comment:
Consider using this version instead https://github.com/awslabs/aws-c-common/blob/master/include/aws/common/stdbool.h

Reply:
Done